test/failed-directories.test.js

```diff
@@ -124,6 +124,36 @@ describe('cosmiconfig', () => {
       ]);
```

Comment:
Could you add a couple of tests for failed search with the default behavior of `ignoreEmpty`?

Reply:
do you think we can just utilize the existing failure tests and toss an empty file in them?  Thanks for catching this btw

Reply:
Yes, that would be great. Tests are already quite verbose and numerous.